### PR TITLE
[fullScreen.js] Deutsch Translation

### PR DESCRIPTION
**Pull request by SupLiCatinG**
Adding deutsch translation for fullScreen.js

### CODE REVIEW

File: Extensions/fullScreen.js [hunk 1]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+            albumRadio: "Abspielen von Alben Radio",
+            playlistRadio: "Abspielen von Playlist Radio",
+        },
+        upnext: "ALS NÄCHSTES",
+        unknownArtist: "Künstler (nicht verfügbar)",
+        settings: {
+            switchToTV: "Wechseln zu TV-Modus",
+            switchToFullscreen: "Wechseln zu Standardmodus",
+            tvModeConfig: "Konfiguration des TV-Modus",
+            fullscreenConfig: "Vollbildmodus konfiguration",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:149]: I'd instead use "Einstellungen des Vollbildmodis" or something like that, "Konfiguration" is a noun, and thus has to be capitalized

File: Extensions/fullScreen.js [hunk 2]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+            exit: "Exit",
+
+            pluginSettings: "Plugin Einstellungen",
+            fsHideOriginal: "Stock-Schaltfläche ausblenden (Spotify Premium)",
+            language: "Sprache",
+
+            lyricsHeader: "Liedtext-Einstellungen",
+            lyrics: "Liedtext",
+            lyricsAlignment: {
+                setting: "Liedtext Ausrichtung",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:159]: maybe "Ausrichtung des Liedtextes"?

File: Extensions/fullScreen.js [hunk 3]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+                left: "Links",
+                center: "Center",
+                right: "Rechts",
+            },
+            lyricsAnimationTempo: "Liedtext Animationtempo",
+
+            generalHeader: "Allgemeine Einstellungen",
+            progressBar: "Fortschrittsanzeige",
+            playerControls: "Wiedegergabe steuerung",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:168]: again, "Steuerung" is a noun, in this case, I'd join the two words together

File: Extensions/fullScreen.js [hunk 4]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+            trimTitle: "Trimmen Titel",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:169]: "Titel trimmen" / "Titel zuschneiden"

File: Extensions/fullScreen.js [hunk 5]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+            trimTitleUpNext: "Trim Title(Up Next)",
+            showAlbum: {
+                setting: "Album anzeigen",
+                never: "Niemals",
+                always: "Jemand",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:174]: "Immer"

File: Extensions/fullScreen.js [hunk 6]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+                date: "Veröffentlichungsdatum anzeigen",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:175]: "Immer + Veröffentlichungsdatum"

File: Extensions/fullScreen.js [hunk 7]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+            fullscreen: "Vollbildmodus",
+
+            extraHeader: "Zusätzliche Funktionalität",
+            backgroundChoice: {
+                setting: "Hintergrundauswahl",
+                color: "Einfarbig",
+                artwork: "Titelbild",
+            },
+            extraControls: "Zusätzliche Steuerelemente",
+            upnextDisplay: "Nächste-titel Anzeige",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:189]: "Nächster Titel-Anzeige"
feel free to improve, my translation here is rather bad

File: Extensions/fullScreen.js [hunk 8]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+            contextDisplay: {
+                setting: "Kontextanzeige",
+                always: "Jemand",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:192]: "Immer"

File: Extensions/fullScreen.js [hunk 9]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+                never: "Niemals",
+                mouse: "Bei Mausbewegung",
+            },
+            volumeDisplay: {
+                setting: "Anzeige der Lautstärkeleiste",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:197]: "Anzeige der Lautstärke" / "Lautstärkenleiste"

File: Extensions/fullScreen.js [hunk 10]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+                always: "Jemand",
+                never: "Niemals",
+                mouse: "Bei Mausbewegung",
+                volume: "Bei Lautstärkeänderung",
+            },
+
+            appearanceHeader: "Erweitert/Aussehen",
+            appearanceSubHeader: "Achtung! Ändern Sie nur, wenn Sie wissen, was Sie tun!",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:205]: "... Ändern Sie **diese Einstellungen** nur, wenn ..."

File: Extensions/fullScreen.js [hunk 11]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+            themedIcons: "Themed Symbole",
+            invertColors: {
+                setting: "IFarben umkehren",
+                never: "Niemals",
+                always: "Jmenad",
+                auto: "Automatisch (basierend auf Hintergrund)",
+            },
+            backAnimationTime: "Background Animation Time",
+            upnextScroll: {
+                setting: "Upnächste Scroll-Animation",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:225]: "Nächstes Lied Scroll-Animation" or something like that maybe

File: Extensions/fullScreen.js [hunk 12]
```diff
@@ -121,6 +121,121 @@ const translations = {
...
+                mq: "Marquee/Scrolling",
+                sp: "Spotify/Translating",
+            },
+            upnextTime: "Upnächste Zeit zu zeigen ",
```
Comment (SIMULATAN) [inline Extensions/fullScreen.js:229]: `"nächster Titel" Anzeigedauer`

**Review by SIMULATAN — CHANGES_REQUESTED** [review]
Must've been a load of work :sweat_smile: 
Just looked through it and added some comments on things that stood out to me.

**Comment by daksh2k** [comment]
Thanks @SupLiCatinG and @SIMULATAN for the translations,
Can you make the changes requested in this pull request, or are you opening another?

**Comment by SupLiCatinG** [comment]
I applied changes to new pull request.

**Pull request closed without merging**